pop-api/src/lib.rs

```diff
@@ -33,6 +33,16 @@ mod constants {
 	pub(crate) const BALANCES: u8 = 10;
 }
 
+/// Helper method to build `ChainExtensionMethod``
```

Comment:
Can we add comments for the arguments please? We can then simply copy them to the version specific functions.